contrib/core/actions/send_mail/send_mail

```diff
@@ -18,7 +20,7 @@ if [[ -z "${BODY// }" && $SEND_EMPTY_BODY == 'True' ]] || [[ -n $BODY ]]; then
 TO: ${TO}
 FROM: ${FROM}
 SUBJECT: ${SUBJECT}
-Content-Type: text/html
+Content-Type: ${CONTENT_TYPE}
 
 ${BODY}
 <br><br>
```

Comment:
Should probably also replace this with "\n" if content type is text/plain.

Besides that, LGTM.